platform/mellanox/smart-switch/dpuctl/dpuctl_hwm.py

```diff
@@ -0,0 +1,195 @@
+"""Class Implementation for per DPU functionality"""
+import os.path
+
+try:
+    from .inotify_helper import InotifyHelper
+    from sonic_py_common.logger import Logger
+    from sonic_platform import utils
+except ImportError as e:
+    raise ImportError(str(e)) from e
+
+EVENT_BASE = "/var/run/hw-management/events/"
+SYSTEM_BASE = "/var/run/hw-management/system/"
+CONFIG_BASE = "/var/run/hw-management/config"
+logger = Logger()
+
+WAIT_FOR_SHTDN = 60
+WAIT_FOR_DPU_READY = 60
+
+
+class MyMessageWriter():
```

Comment:
MyMessageWriter? :)
Please change the name

Reply:
Changed `MyMessageWriter` to `DataWriter`